src/components/institution/UnconfirmedOrganizationBox.tsx

```diff
@@ -0,0 +1,55 @@
+import ErrorIcon from '@mui/icons-material/Error';
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, BoxProps, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { DeleteIconButton } from '../../pages/messages/components/DeleteIconButton';
+import { dataTestId } from '../../utils/dataTestIds';
+import { StyledOrganizationBox } from './OrganizationBox';
+
+interface UnconfirmedOrganizationBoxProps extends Pick<BoxProps, 'sx'> {
```

Comment:
dette kan også fjernes nå
```suggestion
interface UnconfirmedOrganizationBoxProps {
```

Reply:
Nei, det trengs fordi vi sender inn width=100% til den

Reply:
Men så ikke ut som det trengtes når jeg sjekka, kanskje pga `flexGrow`?

Reply:
Det trengtes da jeg sjekka på OrganizationBox ihvertfall.

Reply:
gjør forsåvidt ikke noen skade heller, bare alltid fint å få fjerna overflødig kode